Makefile

```diff
@@ -0,0 +1,36 @@
+.PHONY: all
+all: c csharp go java nim nodejs python rust
+
+.PHONY: c
+c:
+	@make -C src
+
+.PHONY: csharp
+csharp:
+	@make -C bindings/csharp
+
+.PHONY: go
+go:
+	@cd bindings/go && go clean -cache && go test
+
+.PHONY: java
+java:
+	@make -C bindings/java
+
+.PHONY: nim
+nim:
+	@cd bindings/nim && nim test
+
+.PHONY: nodejs
+nodejs:
+	@make -C bindings/node.js
+
+.PHONY: python
+python:
+	@make -C bindings/python
+
+.PHONY: rust
+rust:
+	@cargo test --features generate-bindings
+	@cargo bench --no-run
```

Comment:
I was not expecting to see the benches getting run. I thought this would just build and run tests.
No strong opinion.
(PR LGTM other than this)

Reply:
This doesn't run the benchmarks, just builds them; see the `--no-run` flag.

But this did remind me to disable benchmarks for Java bindings. Thanks!